docusaurus/video/docusaurus/docs/api/basics/get_started.mdx

```diff
@@ -206,10 +250,148 @@ func main() {
 
 ## Updating a call
 
+<Tabs groupId="examples">
+<TabItem value="js" label="JavaScript">
+
+```js
+call.update({ settings_override: { audio: { mic_default_on: true } } });
+
+// or to update custom data
+call.update({ custom: { color: "red" } });
+```
+
+</TabItem>
+<TabItem value="py" label="Python">
+
+```py
+def hello_world():
+    print("Hello, world!")
+```
+
+</TabItem>
+<TabItem value="go" label="Golang">
+
+```go
+TODO
+```
+
+</TabItem>
+</Tabs>
+
 ## Start HLS broadcasting
 
+<Tabs groupId="examples">
+<TabItem value="js" label="JavaScript">
+
+```js
+call.startBroadcasting();
+
+// to end broadcasting
+call.stopBroadcasting();
+```
+
+</TabItem>
+<TabItem value="py" label="Python">
+
+```py
+def hello_world():
+    print("Hello, world!")
+```
+
+</TabItem>
+<TabItem value="go" label="Golang">
+
+```go
+TODO
+```
+
+</TabItem>
+</Tabs>
+
 ## Set up a call for RTMP
 
+<Tabs groupId="examples">
+<TabItem value="js" label="JavaScript">
+
+```js
+const resp = await call.getOrCreate();
+const address = `${resp.call.ingress.rtmp.address}/<API secret>`;
```

Comment:
Is it an upcoming change? Because currently the address already contains the call type, id and api key: `rtmps://video-ingress-us-east-1.stream-io-api.com:1935/hd8szvscpxvd/default/X8hXgT6uXgfH`

Reply:
Is not upcoming, it is already in this way, we should merge the PR but right now works as I posted